### DISCUSSION TITLE
Connect to jupyter through VM

### DISCUSSION THREAD

**Discussion by konsweb-122** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#68] [3.0 VMs and data (Azure VM, local deployment, and Jupyter)]
May i know how can i solve this error? I cant open jupyter notebook from my VM. 

jupyter_1   | [W 16:42:37.944 NotebookApp] No web browser found: could not locate runnable browser.

**Comment by EY-glowglowglow** [comment]
Hi there,
I'd suggest your get your error logs following the instructions [here ](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/4a.-Troubleshooting:-FAQ)so that we can better understand the issue.

**Reply by konsweb-122** [reply]
I am unable to download the log
ubuntu@CubeInABox:~$ scp ubuntu@13.75.138.9:/var/log/install-cube.log %USERPROFILE%/Downloads
ubuntu@13.75.138.9: Permission denied (publickey).

**Reply by EY-glowglowglow** [reply]
@codeindulgence  any ideas on how to solve this?

**Reply by nickbutler** [reply]
Hi @konsweb-122. It looks like you're running this command on the VM itself, you should be running it from your local machine.

Also, please obvserve my other answer to this topic.

Cheers

**Comment by nickbutler** [comment]
Hi @konsweb-122, this is expected to happen, since Jupyter is running in a server environment where there is no browser.

In a local environment, Jupyter will automatically open in your browser when started, but cannot do this from a server, hence the warning issued.

Cheers
